### PR TITLE
[FIX] sale,stock,website_sale: DHL Connector

### PR DESCRIPTION
**Pull request by svs-odoo**
Renames "DHL", "DHL USA" and "DHL Express" into "DHL Express Connector".

task-2581265

### CODE REVIEW

**Comment by amoyaux** [comment]
robodoo r+

**Comment by amoyaux** [comment]
robodoo r+

**Comment by svs-odoo** [comment]
@fw-bot up to 15.0

File: addons/website_sale/i18n/website_sale.pot [hunk 1]
```diff
@@ -988,7 +988,7 @@ msgstr ""
 
 #. module: website_sale
 #: model_terms:ir.ui.view,arch_db:website_sale.res_config_settings_view_form
-msgid "DHL"
+msgid DHL Express Connector
```
Comment (MiquelRForgeFlow) [inline addons/website_sale/i18n/website_sale.pot:991]:
```diff
- DHL Express Connector
+ "DHL Express Connector"
```
right?

Comment (svs-odoo) [reply addons/website_sale/i18n/website_sale.pot:991]: Damn it, you're right :grimacing:

**Pull request closed without merging**